api/lib/opentelemetry/context.rb

```diff
@@ -29,23 +33,50 @@ def current
         Thread.current[KEY] ||= ROOT
       end
 
-      # Sets the current context
-      #
-      # @param [Context] ctx The context to be made active
+      # @api private
       def current=(ctx)
         Thread.current[KEY] = ctx
       end
 
+      # Associates a Context with the caller's current Fiber. Every call to
+      # this operation should be paired with a corresponding call to detach.
+      #
+      # Returns a token to be used with the matching call to detach
+      #
+      # @param [Context] context The new context
+      # @return [Integer] A token to be used when detaching
```

Comment:
I think we should be vague here.
```suggestion
      # @return [Object] A token to be used when detaching
```